app/views/home/index.html.erb

```diff
@@ -1,2 +1,60 @@
-<h1>Home#index</h1>
-<p>Find me in app/views/home/index.html.erb</p>
+<%= content_tag :main,
+								data: { controller: "home" } do %>
+	<!-- FEATURED -->
+	<section class="featured">
+		<div class="container">
+			<div class="common__title common__title--center">
+				Featured
+			</div>
+			<div class="featured__carousel js-featured__carousel">
+				<% @books.each do |book| %>
+					<div class="featured__container">
+						<div class="featured__wrapper">
+							<div class="featured__books books">
+								<div class="featured__thumb-wrapper books__thumb">
+									<div class="featured__thumb books__thumb--front">
+										<%= link_to "http://google.com", url_target: "blank" do %>
+											<%= image_tag book.images.first.variant(:featured), alt: book.title %>
+										<% end %>
+									</div>
+									<div class="books__modules">
+										<% if current_user && current_user.cart.exists_book_in_cart?(book.id) %>
```

Comment:
Do we have N+1 query this way?

Reply:
Yes, we do. I have an idea to resolve this issue. I will get all book ids (array of book_id) in the current user's cart and check the book (in each loop) is in this array or not. I try to implement this solution and it works. Could you give me your opinion about my solution for this case?

Reply:
I think that solution is fine